### ISSUE TITLE
Prevent close toolbar after prop changed

### ISSUE THREAD

**Issue by SemyonL95** [FranzSkuffka/vue-medium-editor#63]
After i update text prop, Medium editor toolbar is closing, i don't understand how to prevent it, help please

**Comment by janwirth** [comment]
Please provide a minimal working example.

**Comment by SemyonL95** [comment]
i found a problem, but don't know how to solve it, when i pass a prop - option it closes after update, if not it doesn't

**Comment by SemyonL95** [comment]
```
import editor from 'vue2-medium-editor'

const collection = {
  item1: 'hello wold 1',
}

export default {
  name: 'app',
  components: {editor},
  render (h) {
    return h('editor', {
      props: {
        text: this.text.item1,
        // options: {
        //   toolbar: {
        //     buttons: ['bold']
        //   }
        // }
      },
      on: {edit: this.onEdit}
    })
  },

  data () {
    return {
      text: collection
    }
  },
  
  methods: {
    onEdit (obj) {
      console.log(obj)
      this.text.item1 = obj.event.target.innerHTML
    }
  }
}
```

if uncomment options and pass emty or not empty object, anything, to prop options - toolbar starts hide after update prop text.
I think, for some cause it's recreate instance of editor because, id in DOM of Medium editor increase after each update

**Comment by SemyonL95** [comment]
i found what problem is.
This happening cause, watcher in editor component which watch for options prop. So i added just comparing of new options object, and old, and just reload if they are not same, i made a pull request #64

**Comment by janwirth** [comment]
Thank you for supporting the project. I released 1.1.2 which should fix this issue.

Next time make sure your PR is cleaner.
